Application/Dtos/Identity/LoginDto.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Core.Entities;
+namespace Application.Dtos.Identity
+{
+    public class LoginDto
```

Comment:
Abbreviations should be named with CamelCase
```suggestion
    public class LoginDTO
```